docs/tas/datacenter/99-unlabel-nodes.sh

```
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Removes the labels added by 00-label-nodes.sh.
+set -euo pipefail
+
+nodes=($(oc get nodes -l cloud.provider.com/node-group=tas-group -o name))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)